ui-admin/src/study/kits/KitEnrolleeSelection.tsx

```diff
@@ -49,11 +50,23 @@ export default function KitEnrolleeSelection({ studyEnvContext }: { studyEnvCont
   const [columnVisibility, setColumnVisibility] = useState<VisibilityState>({})
   const [columnFilters, setColumnFilters] = useState<ColumnFiltersState>([
     { id: 'consented', value: true },
-    { id: 'kitRequested', value: false },
     { id: 'requiredSurveysComplete', value: true }
   ])
+
   const [showRequestKitModal, setShowRequestKitModal] = useState(false)
 
+  const { isLoading: isLoadingKitTypes } = useLoadingEffect(async () => {
```

Comment:
I think it will keep the mental model of the page simpler if we include this Api call in the useLoadingEffect below and use a Promise.all to wait on both calls